src/components/Voting.js

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from "react";
+import { PatchArticleVotes } from "./api-logic";
+
+const Voting = ({ singleArticle, article_id }) => {
+  const [vote, setVote] = useState(0);
+  const [disabledButton, setDisabledButton] = useState(false);
+
+  useEffect(() => {
+    PatchArticleVotes(article_id, vote)
+      .then((response) => response.json())
+      .catch((error) => {
+        return alert(
```

Comment:
This works - but ewww to alerts and pop-ups. I'd keep it for now, but look at refactoring this into an error message that displays on screen under the voting buttons. Just some text in <p> tags would work 👍